samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c

```diff
@@ -107,32 +107,92 @@ static float sqrt(float square)
 	return root;
 }
 
+static s32_t ceiling(float num)
+{
+	s32_t inum;
+
+	inum = (s32_t) num;
+	if (num == (float) inum) {
+		return inum;
+	}
+
+	return inum + 1;
+}
+
+static bool constrain_lightness_actual_state(u16_t var)
+{
+	bool is_value_within_range;
+
+	is_value_within_range = false;
+
+	if (var > 0 && var <
+		light_lightness_srv_user_data.lightness_range_min) {
+		var =
+		light_lightness_srv_user_data.lightness_range_min;
```

Comment:
I'd suggest looking for shorter naming of your variables, if you have to do this kind of line splitting for only two tabs deep indentation.

Reply:
If I shorter the name that will reduce readability. I will go with your first suggestion

Reply:
Ok....going to modifiy **struct light_lightness_state** as ...

struct light_lightness_state {
	u16_t linear;
	u16_t actual;
	u16_t last;
	u16_t def;

	u8_t status_code;
	u16_t light_range_min;
	u16_t light_range_max;

	u8_t last_tid;
	u16_t last_tx_addr;
};

Reply:
done !!